projects/angular-showcase/src/app/business/business-examples/status-showcase/status-showcase.component.html

```diff
@@ -54,18 +66,9 @@ <h3>Status Extended used in table</h3>
   <tbody>
     <tr *ngFor="let row of rows">
```

Comment:
Having a dynamic table for an example with four entries is a bit overkill.
I'd prefer to just have the four rows.
This is also better for consumer who would like to see the source of the example.